src/task_queue/events_nudge/cadmin_events_nudge.py

```diff
@@ -205,7 +213,8 @@ def send_events_nudge():
 
 def send_events_report(name, email, event_list):
     try:
-        change_preference_link = ADMIN_URL_ROOT+"/admin/profile/preferences"
+        cred = encode_data({"email": email})
```

Comment:
Can you move this duplicated code to a utility routine for example add_credentials()?  I think we will be using it a number of links.